addons/scoreboard/mergedplayer.lua

```diff
@@ -234,8 +234,21 @@ function MergedPlayer:wsavg()
     end
 end
 
-
-
+function MergedPlayer:wsacc()
+    local hits, misses = 0, 0
+    
+    for _, p in ipairs(self.players) do
+        for _ in pairs(p.ws) do hits = hits + 1 end
```

Comment:
`table.length` is a non-standard function that is available through our libraries, which does exactly that :) Everyone gets tired of writing loops just to count the length of an array. Just part of Lua's weirdness.

From `Windower/addons/libs/tables.lua`:
```lua
-- Returns the number of elements in a table.
function table.length(t)
    local count = 0
    for _ in pairs(t) do
        count = count + 1
    end

    return count
end
```

If this ever gets refactored, it should probably be changed to use [`lists`](https://github.com/Windower/Lua/blob/dev/addons/libs/lists.lua), which keep a running count and can get the length in constant time. But that's not your job, just mentioning it for possible future maintainers.